package.json

```diff
@@ -183,7 +183,7 @@
     "cozy-realtime": "3.11.0",
     "cozy-scripts": "5.2.0",
     "cozy-stack-client": "23.0.0",
-    "cozy-ui": "50.1.0",
+    "cozy-ui": "50.5.0",
```

Comment:
To be exact this is not "chore". chore is only used for CI stuff or else that doesn't affect production code. 

Here we update a dependency, we modify the code used in production. It should be tagged as feat or fix.

Reply:
well seen, it's done